jobs/sql/extractions/pregnancy_program.sql

```diff
@@ -196,12 +201,21 @@ update temp_pregnancy_program set hiv_status = 'Unknown' where hiv_status is nul
 
 update temp_pregnancy_program set trimester_enrolled = temp_program_obs_latest_value_coded_name(patient_program_id, 'PIH', '11661');
 
+update temp_pregnancy_program t
+inner join encounter e on e.encounter_id  = latest_labor_delivery_summary_encounter_id
+set  latest_labor_delivery_summary_datetime  = e.encounter_datetime;
+
 update temp_pregnancy_program set num_previous_anc_visits = temp_program_obs_latest_value_numeric(patient_program_id, 'CIEL', '1590');
 update temp_pregnancy_program set num_previous_anc_visits = 0 where num_previous_anc_visits is null;
 update temp_pregnancy_program set num_previous_anc_visits = (num_previous_anc_visits - 1) where num_previous_anc_visits > 0;
 
-update temp_pregnancy_program set total_anc_initial = temp_program_encounter_count(patient_program_id, @ancIntake);
-update temp_pregnancy_program set total_anc_followup = temp_program_encounter_count(patient_program_id, @ancFollowup);
+set @postpartum_concept_id = concept_from_mapping('PIH','13137');
+select program_workflow_state_id into @postpartum_state from program_workflow_state where concept_id = @postpartum_concept_id;
+update temp_pregnancy_program set post_partum_state_date = temp_program_earliest_patient_state_date(patient_program_id ,@postpartum_state );
```

Comment:
Try to keep the spacing consistent, note where your commas, parenthesis, and spaces are here and compare with elsewhere.